blocks/OSC/src/OscListener.cpp

```diff
@@ -132,6 +132,13 @@ void OscListener::ProcessMessage( const ::osc::ReceivedMessage &m, const IpEndpo
 			message->addFloatArg(arg->AsFloatUnchecked());
 		else if (arg->IsString())
 			message->addStringArg(arg->AsStringUnchecked());
+		else if(arg->IsBlob()){
```

Comment:
This file seems to have inconsistent style but we might as well be consistent with the `for` loop body and add a space between `else if` and `(arg-> …`